.github/workflows/ci.yaml

```
@@ -0,0 +1,39 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '**'
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+      - name: Setup Golang
+        uses: actions/setup-go@c4a742cab115ed795e34d4513e2cf7d472deb55f # v3.3.1
+        with:
+          go-version-file: go.mod
+      - name: Dependencies
+        run: make deps
+      - name: Test
+        run: make testacc
+      - name: Lint
+        run: make lint
+  release:
```

Shouldn't this job depend on `test`?
```suggestion
  release:
    needs: [ test ]
```

---

It can. We were not placing this dependency on the other migrated OSS projects but I'll add it.
Accepting suggestion.